R/acquisition.R

```diff
@@ -160,13 +160,13 @@ predict.exp_improve <- function(object, new_data, maximize, iter, best, ...) {
 #' @rdname prob_improve
 conf_bound <- function(kappa = 0.1) {
   if (!is.numeric(kappa) & !is_function(kappa)) {
-    rlang::abort("`kappa` should be a number or a function.")
+    cli::cli_abort("{.arg kappa} should be a number or a function.")
   }
   lab <- "the confidence bound"
   if (rlang::is_function(kappa)) {
     farg <- names(formals(kappa))
     if (length(farg) == 0) {
-      rlang::abort("The `trade_off` function should have at least one argument.")
+      cli::cli_abort("The {.fn trade_off} function should have at least one argument.")
```

Comment:
```suggestion
      cli::cli_abort("The {.fn kappa} function should have at least one argument.")
```

I _think_ this error should have referenced this argument all along?

Reply:
Seeing that it's maybe ambiguous whether to use `.fn` or `.arg` in these—fine with me to go the route you've chosen.